CoreScriptsRoot/Modules/Settings/SettingsHub.lua

```diff
@@ -984,15 +998,34 @@ local function CreateSettingsHub()
 			end
 		end)
 	end
+	local function disableVR()
+		this.ClippingShield.Parent = RobloxGui
+		this.Shield.Parent.ClipsDescendants = true
+		this.VRShield.Visible = false
+		this:ShowShield()
+
+		if vrMenuOpened then
+			vrMenuOpened:disconnect()
+			vrMenuOpened = nil
+		end
+		if vrMenuClosed then
+			vrMenuClosed:disconnect()
+			vrMenuClosed = nil
+		end
+
+		local Panel3D = require(RobloxGui.Modules.VR.Panel3D)
+		local panel = Panel3D.Get(thisModuleName)
+		panel:SetVisible(false)
+	end
 
 	local UISChanged;
 	local function OnVREnabled(prop)
-		if prop == "VREnabled" and UserInputService.VREnabled then
-			VREnabled = true
-			enableVR()
-			if UISChanged then
-				UISChanged:disconnect()
-				UISChanged = nil
+		if prop == "VREnabled" then
+			VREnabled = UserInputService.VREnabled 
```

Comment:
It seemed like you removed references to this variable, assuming you removed all of them we should remove this variable as well